src/api.js

```diff
@@ -48,6 +48,13 @@ export const createNewUser = async user => {
   }
 };
 
+export const getSearchResults = async input => {
+  const response = await axios.get(
+    `http://localhost:8080/users/search/${input}`
```

Comment:
You would probably want to pass the input in as a query parameter, not as an id

Reply:
Do you mean pass it in the request body not as part of the url?

Reply:
No, have a read of query strings https://stackoverflow.com/questions/39266970/what-is-the-difference-between-url-parameters-and-query-strings